x/docs/internal/keeper/querier.go

```diff
@@ -58,3 +59,37 @@ func queryGetSentDocuments(ctx sdk.Context, path []string, keeper Keeper) (res [
 
 	return bz, nil
 }
+
+func queryGetReceivedDocsReceipts(ctx sdk.Context, path []string, keeper Keeper) (res []byte, err sdk.Error) {
+	addr := path[0]
+	address, _ := sdk.AccAddressFromBech32(addr)
+
+	var uuid string
+	if len(path) == 2 {
+		uuid = path[1]
+	}
+
+	var receipts []doctypes.DocumentReceipt
+	var bz []byte
+	var err2 error
+
+	//If user wants all his receipts
+	if uuid == "" {
+		receipts = keeper.GetUserReceivedReceipts(ctx, address)
+
+		if receipts == nil {
+			receipts = make([]doctypes.DocumentReceipt, 0)
+		}
+		bz, err2 = codec.MarshalJSONIndent(keeper.cdc, &receipts)
+
+	} else {
+		receipts = keeper.GetUserReceivedReceiptsForDocument(ctx, address, uuid)
+		bz, err2 = codec.MarshalJSONIndent(keeper.cdc, &receipts)
```

Comment:
This line looks identical to like 83. Can we take them out of the `if else` and put them after the `else`?